### PR TITLE
Adding support for TensorFlow 2.x

### PR DESCRIPTION
**Pull request by ifielker**
Hey there! So you want to contribute to a Firebase SDK? 
Before you file this pull request, please read these guidelines:

### Discussion

  * Read the contribution guidelines (CONTRIBUTING.md).
  * If this has been discussed in an issue, make sure to link to the issue here. 
    If not, go file an issue about this **before creating a pull request** to discuss.

### Testing

  * Make sure all existing tests in the repository pass after your change.
  * If you fixed a bug or added a feature, add a new test to cover your code.

### API Changes

  * At this time we cannot accept changes that affect the public API.  If you'd like to help 
    us make Firebase APIs better, please propose your change in an issue so that we 
    can discuss it together.

### CODE REVIEW

File: tests/test_mlkit.py [hunk 1]
```diff
@@ -763,7 +763,12 @@ def teardown_class(cls):
         testutils.cleanup_apps()
 
     @staticmethod
-    def _url(project_id, model_id):
+    def _update_url(project_id, model_id):
+        update_url = 'projects/{0}/models/{1}?updateMask=state.published'
+        return BASE_URL + update_url.format(project_id, model_id)
```
Comment (hiranya911) [inline tests/test_mlkit.py]: Nit: Consider doing the `format()` on the previous line.

File: firebase_admin/mlkit.py [hunk 2]
```diff
@@ -495,12 +497,30 @@ def from_tflite_model_file(cls, model_file_name, bucket_name=None, app=None):
...
+                              .format(tf.version.VERSION))
+
+    @staticmethod
+    def _tf_convert_from_saved_model(saved_model_dir):
+        # Same for both v1.x and v2.x
+        converter = tf.lite.TFLiteConverter.from_saved_model(saved_model_dir)
+        return converter.convert()
+
+    @staticmethod
+    def _tf_convert_from_keras_model(keras_model):
```
[the remark this answers is not in the document]
Comment (ifielker) [reply firebase_admin/mlkit.py]: done

File: firebase_admin/mlkit.py [hunk 3]
```diff
@@ -852,12 +868,12 @@ def create_model(self, model):
 
     def update_model(self, model, update_mask=None):
         _validate_model(model, update_mask)
-        data = {'model': model.as_dict(for_upload=True)}
+        path = 'models/{0}'.format(model.model_id)
         if update_mask is not None:
-            data['updateMask'] = update_mask
+            path = path + '?updateMask={0}'.format(update_mask)
```
[the remark this answers is not in the document]
Comment (ifielker) [reply firebase_admin/mlkit.py]: update mask is 'state.published' or empty.

File: firebase_admin/mlkit.py [hunk 4]
```diff
@@ -884,15 +900,20 @@ def list_models(self, list_filter, page_size, page_token):
...
-            payload['list_filter'] = list_filter
+            params['filter'] = list_filter
         if page_size:
-            payload['page_size'] = page_size
+            params['page_size'] = page_size
         if page_token:
-            payload['page_token'] = page_token
+            params['page_token'] = page_token
+        path = 'models'
+        if params != {}:
```
[the remark this answers is not in the document]
Comment (ifielker) [reply firebase_admin/mlkit.py]: done

File: tests/test_mlkit.py (hunk 1 above)
Comment (ifielker) [reply tests/test_mlkit.py]: it's one character too long. I can wrap it or I can leave it? Which do you prefer?

Comment (hiranya911) [reply tests/test_mlkit.py]: May be something like:

```
update_url = 'projects/{0}/models/{1}?updateMask=state.published'.format(
    project_id, model_id)
```

**Pull request merged**